Source/WebCore/rendering/BackgroundPainter.cpp

```diff
@@ -485,14 +485,18 @@ void BackgroundPainter::paintFillLayer(const Color& color, const FillLayer& bgLa
         if (!geometry.destinationRect.isEmpty() && (image = bgImage->image(backgroundObject ? backgroundObject : &m_renderer, geometry.tileSize, isFirstLine))) {
             context.setDrawLuminanceMask(bgLayer.maskMode() == MaskMode::Luminance);
 
+            // FIXME: Allow HDR display for background images when CSS HDR images are able to set GraphicsLayer::drawHDRContent.
+            auto headroom = Headroom::None;
```

Comment:
There's also CSS image drawing in `BorderPainter::paintBorder`, and probably more; look for calls to `paintNinePieceImage`

Reply:
The HDR display is disabled in `NinePieceImage::paint()`